videohub_client/management/commands/migrate_channel_id.py

```diff
@@ -36,16 +38,20 @@ def handle(self, *args, **options):
 
             if resp.ok:
                 channel = resp.json()['channel']
-                print('{} {} --> {} {}'.format(video.id,
-                                               video.title.strip().encode('utf-8'),
-                                               channel['id'],
-                                               channel['name'].encode('utf-8')
-                                               ))
+                print('[{}] {} --> [{}] {}'.format(video.id,
+                                                   video.title.strip().encode('utf-8')[:50],
+                                                   channel['id'],
+                                                   channel['name'].encode('utf-8')
+                                                   ))
                 if not options['check']:
                     video.channel_id = channel['id']
                     video.save()
             else:
-                raise CommandError('Video request failed: {} --> {} {}'.format(
+                self.stderr.write(self.style.ERROR('Video request failed: {} --> {} {}'.format(
                     url,
                     resp.status_code,
-                    resp.reason))
+                    resp.reason)))
+                failed = True
+
+        if failed:
+            raise CommandError('Failed to update 1 or more videos')
```

Comment:
if you want, you could have some kind of array of failed videos and print that out, so you can check why they failed.